atomic.h: No such file or directory with esp32s3 of espidf platform

### LVGL version

v9.5.0

### Platform

ESP32s3 on platformio with espidf framework and `#define LV_USE_OS   LV_OS_FREERTOS`

### What happened?

`src/osal/lv_freertos.c:18:10: fatal error: atomic.h: No such file or directory`

espidf places freertos headers in `freertos/`, it looks like theres a check in `lv_freertos.h` for ESP32 platform, but not in `lv_freertos.c`

### How to reproduce?

_No response_

--- a/src/osal/lv_freertos.c
+++ b/src/osal/lv_freertos.c
@@ -15,7 +15,11 @@
 #include "lv_os_private.h"
 #if LV_USE_OS == LV_OS_FREERTOS
 
+#ifdef ESP_PLATFORM
+#include "freertos/atomic.h"
+#else
 #include "atomic.h"
+#endif
 
 #include "../tick/lv_tick.h"
 #include "../misc/lv_log.h"

--- a/src/osal/lv_freertos.c
+++ b/src/osal/lv_freertos.c
@@ -16,9 +16,9 @@
 #if LV_USE_OS == LV_OS_FREERTOS
 
 #ifdef ESP_PLATFORM
-#include "freertos/atomic.h"
+    #include "freertos/atomic.h"
 #else
-#include "atomic.h"
+    #include "atomic.h"
 #endif
 
 #include "../tick/lv_tick.h"

